src/main/scala/org/broadinstitute/dsde/firecloud/model/ESMapping.scala

```diff
@@ -1,7 +1,81 @@
 package org.broadinstitute.dsde.firecloud.model
```

Comment:
I love the consolidation, but now the name of this file is misleading - it's much more than just the mapping.